VocaDbWeb.Core/Controllers/Api/ResourcesApiController.cs

```diff
@@ -0,0 +1,121 @@
+#nullable disable
+
+using System.Buffers;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Resources;
+using AspNetCore.CacheOutput;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using VocaDb.Web.Resources.Domain.ReleaseEvents;
+using ApiController = Microsoft.AspNetCore.Mvc.ControllerBase;
+
+namespace VocaDb.Web.Controllers.Api
+{
+	/// <summary>
+	/// Loads localized string resources.
+	/// </summary>
+	[Route("api/resources")]
+	[DefaultCasingConfig]
+	[ApiController]
+	public class ResourcesApiController : ApiController
+	{
+		// Code from: https://stackoverflow.com/questions/56127510/aspnet-core-input-output-json-serialization-settings-at-controller-level/56127866#56127866
+		class DefaultCasingConfig : ActionFilterAttribute
+		{
+			public override void OnResultExecuting(ResultExecutingContext context)
+			{
+				if (context.Result is ObjectResult objectResult)
+				{
+					var serializerSettings = new JsonSerializerSettings
+					{
+						ContractResolver = new DefaultContractResolver(),
+					};
+					// Code from: https://github.com/aspnet/Mvc/issues/8128#issuecomment-407214518
+					var options = context.HttpContext.RequestServices.GetRequiredService<IOptions<MvcOptions>>();
+					var mvcOptions = options.Value;
+					var jsonFormatter = new NewtonsoftJsonOutputFormatter(serializerSettings, ArrayPool<char>.Shared, mvcOptions);
+					objectResult.Formatters.Add(jsonFormatter);
+				}
+			}
+		}

```

Comment:
Implemented `DefaultCasingConfig`. It seems that `ResourcesApiController` uses PascalCase rather than camelCase.

The JSON response with `DefaultCasingConfig` (See https://vocadb.net/api/resources/en-US/?setNames%5B%5D=albumSortRuleNames):

```json
{
   "albumSortRuleNames":{
      "RatingAverage":"Rating average",
      "RatingTotal":"Total score",
      "ReleaseDate":"Release date",
      "CollectionCount":"Collection count",
      "AdditionDate":"Addition date",
      "Name":"Name"
   }
}
```

The JSON response without `DefaultCasingConfig`:

```json
{
   "albumSortRuleNames":{
      "ratingAverage":"Rating average",
      "ratingTotal":"Total score",
      "releaseDate":"Release date",
      "collectionCount":"Collection count",
      "additionDate":"Addition date",
      "name":"Name"
   }
}
```

Reply:
I guess there was a reason ResourceApi was left PascalCase. But yeah, default should be camelCase.

Reply:
Added `[Obsolete]` to `DefaultCasingConfig`.